monkey/infection_monkey/monkey.py

```diff
@@ -243,6 +243,11 @@ def close_tunnel():
 
     @staticmethod
     def self_delete():
+        if utils.remove_monkey_dir():
```

Comment:
Instead of all this code, just do:
```suggestion
        status = utils.remove_monkey_dir() ? ScanStatus.USED : ScanStatus.SCANNED
        T1107Telem(status , utils.get_monkey_dir_path()).send()
```